db/seeds.rb

```diff
@@ -5,3 +5,13 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+require 'open-uri'
+
+page = Nokogiri::HTML(open("http://www.lesson1.ru/vocabulary/lexicographer/food.html"))
+trs = page.css('table.tbl3 tr:not(.first)')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.